packages/ui/ProgressBar/ProgressBar.js

```diff
@@ -72,8 +72,9 @@ const ProgressBar = kind({
 		 * The following classes are supported:
 		 *
 		 * * `progressBar` - The root component class
-		 * * `fill`        - The foreground node of the progress bar
-		 * * `load`        - The background node of the progress bar
+		 * * `bar`         - The background node (the empty part of the progressBar)
```

Comment:
We should note the addition of this in the change log

Reply:
I suppose we could. It didn't seem big enough for that.

Reply:
As public API, I think it is important we manage these with the same rigor as other public APIs.